imagedumper_desktop.go

```diff
@@ -34,7 +36,7 @@ func availableFilename(prefix, postfix string) (string, error) {
 	name := fmt.Sprintf("%s%s%s", prefix, now.Format(datetimeFormat), postfix)
 	for i := 1; ; i++ {
 		if _, err := os.Stat(name); err != nil {
```

Comment:
```go
		if runtime.GOOS != "js" {
			if _, err := os.Stat(name); err != nil {
				if os.IsNotExist(err) {
					break
				}
				if !os.IsNotExist(err) {
					return "", err
				}
			}
		}
```

Or, `name` keeps an empty string on browsers, right?

Reply:
checking for ENOSYS just ignores the error on any platform that doesn't implement it. either way works for me, but name shouldn't be an empty string

Reply:
Ah, you are right, `name` is not an empty. NVM